tests/Doctrine/Migrations/Tests/Tools/Console/Command/DiffCommandTest.php

```diff
@@ -8,6 +8,7 @@
 use Doctrine\Migrations\DependencyFactory;
 use Doctrine\Migrations\Generator\ClassNameGenerator;
 use Doctrine\Migrations\Generator\DiffGenerator;
+use Doctrine\Migrations\MigrationPlanCalculator;
```

Comment:
This should fix static analysis:

```suggestion
use Doctrine\Migrations\Version\MigrationPlanCalculator;
```

You can run phpstan locally by running `vendor/bin/phpstan analyse` to see if it worked without pushing.